notebook/notebookapp.py

```diff
@@ -239,16 +239,21 @@ def init_handlers(self, settings):
         handlers.extend(load_handlers('services.security.handlers'))
         
         # BEGIN HARDCODED WIDGETS HACK
+        widgets = None
         try:
-            import ipywidgets
+            widgets = __import__('jupyter-js-widgets-nbextension')
+        except:
+            try:
+                widgets = __import__('ipywidgets')
+            except:
+                app_log.warning('jupyter-js-widgets-nbextension package not installed.  Widgets are unavailable.')
```

Comment:
Hum... i'm not sure, because people **should** be able to install only an old version of ipywidgets. 
I think the error message should reference both just in case.